README.md

```diff
@@ -238,6 +238,13 @@ name. If you have multiple sites on the same host, these must be unique.
 COMPOSE_PROJECT_NAME=isle-site-template
 ```
 
+ISLANDORA_TAG tells Docker what version of [Isle Buildkit](https://github.com/Islandora-Devops/isle-buildkit) 
+to use for the images. You should set this to the most 
+[recent release](https://github.com/Islandora-Devops/isle-buildkit/releases) number, 
+unless you have a reason to use older images.
+
+> Note: You should not use `ISLANDORA_TAG=main` in production.
```

Comment:
think this should have a bit more color around it [using a highlighter](https://github.com/orgs/community/discussions/16925)

```suggestion
> [!WARNING]
> You should not use `ISLANDORA_TAG=main` in production.
```